MdeModulePkg/Include/Guid/SmBiosTableHob.h

```diff
@@ -0,0 +1,20 @@
+/** @file
+ Define the GUID gPldSmbios3TableGuid and gPldSmbiosTableGuid HOB struct.
+
+Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _SMBIOS_TABLE_HOB_H_
+#define _SMBIOS_TABLE_HOB_H_
+
+#include <Uefi.h>

```

Comment:
Is it ok to just include <Base.h>?
We don't want to let PLD interface depend on UEFI types.